html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <div>
+    Email
+    <br />
```

Comment:
Hi Henry! Please only use break statements when trying to divide paragraphs onto new lines. To move form elements onto new lines, please use divs.